crates/typst-library/src/foundations/array.rs

```diff
@@ -815,6 +815,17 @@ impl Array {
     ///
     /// Returns an error if two values could not be compared or if the key
     /// function (if given) yields an error.
+    /// 
+    /// The key function can return a tuple to sort by multiple keys.
```

Comment:
```suggestion
    /// The key function can return a tuple to sort according to the lexicographic order.
```
The exact phrasing doesn't matter, but we should probably specify that the lexicographic order is used.